system/settings/settings.xml

```diff
@@ -1119,6 +1065,170 @@
             <show more="true" details="true">installed</show>
           </control>
         </setting>
+        <setting id="musiclibrary.overridetags" type="boolean" label="20220" help="20221">
+          <level>1</level>
+          <default>false</default>
+          <control type="toggle" />
+        </setting>
+      </group>
+      <group id="3" label="39120">
+        <setting id="musiclibrary.preferonlinealbumart" type="boolean" label="20224" help="20225">
+          <level>1</level>
+          <default>false</default>
+          <control type="toggle" />
+        </setting>
+		<!-- Hidden setting indicating music art settings have been migrated from old advancedsettings.xml format-->
+        <setting id="musiclibrary.artsettings" type="boolean" label="0" help="">
+          <level>4</level>
+          <default>false</default>
+          <control type="toggle" />
+        </setting>
+		<setting id="musiclibrary.artworklevel" type="integer" label="39137" help="39138">
+          <level>1</level>
+          <default>0</default>
+          <constraints>
+            <options>
+              <option label="39140">0</option> <!-- maximum -->
+              <option label="39141">1</option> <!-- minimum -->
+              <option label="39142">2</option> <!-- user configured -->
+            </options>
+          </constraints>
+          <control type="list" format="string" />
+        </setting>
+        <setting id="musiclibrary.usealllocalart" type="boolean" label="39127" help="39128">
+          <level>1</level>
+          <default>false</default>
+          <control type="toggle" />
+		  <dependencies>
+            <dependency type="visible" setting="musiclibrary.artworklevel" operator="is">2</dependency>
+          </dependencies>
+        </setting>
+        <setting id="musiclibrary.useallremoteart" type="boolean" label="39129" help="39130">
+          <level>1</level>
+          <default>false</default>
+          <control type="toggle" />
+		  <dependencies>
+            <dependency type="visible" setting="musiclibrary.artworklevel" operator="is">2</dependency>
+          </dependencies>		  
```

Comment:
there's trailing whitespace here and on a few other lines.
also, the indenting is a bit off on several lines.

Reply:
Darn whitespace. I use Notepad+ to edit *.xml and it leaves spaces places. Is there some tool that does the right format?